docs/communicate-with-javascript.md

```diff
@@ -1034,6 +1040,8 @@ JavaScript objects are used in a variety of use cases:
 - As a class.
 - As a module to import/export.
 
+<!-- Mention that Everything in JavaScript is an object -->
```

Comment:
Can you elaborate? Some things in JS are not an object: strings, numbers, booleans,... are primitives:

```js
let str = "neal";
str.age = 22;
console.log(str.age);
```